Model/Order/OrderAddress.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace ScandiPWA\SalesGraphQl\Model\Order;
+
+use Magento\Sales\Api\Data\OrderAddressInterface;
+use Magento\Sales\Api\Data\OrderInterface;
+use Magento\SalesGraphQl\Model\Order\OrderAddress as CoreOrderAddress;
+
+/**
+ * Class to get the order address details
+ */
+class OrderAddress extends CoreOrderAddress
+{
+    /**
+     * Get the order Shipping address
+     *
+     * @param OrderInterface $order
+     * @return array|null
+     */
+    public function getOrderShippingAddress(
+        OrderInterface $order
+    ): ?array {
+        $shippingAddress = null;
+        if ($order->getShippingAddress()) {
```

Comment:
newlines before if and return's(as well as foreaches, try's, etc) are...well, not a mandatory code style, but highly recommended (unless it is the only contents of the code block or beginning of the code block)

if not using that, then at least sticking to same style across the code (as some places have newlines and some don't)